### PR TITLE
PR for resolving issue #1056

### PR DESCRIPTION
**Pull request by navneetkarnam**
- This PR fixes issue #1056 
- I have added a note in step 8 as to where the survey appears.

### CODE REVIEW

**Comment by kanishk1010** [comment]
@navneetkarnam @tjyothsna16 made some changes. Take a look. 
@navneetkarnam Please add a rawgit link to your PR

**Comment by navneetkarnam** [comment]
* @sarah-lu102 @kanishk1010 I have no idea why my rawgit link isn't working. Any suggestions as to what might be wrong? This is the link btw ``https://rawgit.com/navneetkarnam/navneetkarnam.github.io/navneetkarnam-issue-#1056/#!pages/firststeps.md``

**Comment by sarah-lu102** [comment]
I found the problem- if you go to your branch on github ( this link https://github.com/navneetkarnam/navneetkarnam.github.io/tree/navneetkarnam-issue-%231056), the branch name is `navneetkarnam-issue-%231056`, not `navneetkarnam-issue-#1056`. 

This rawgit link works: https://rawgit.com/navneetkarnam/navneetkarnam.github.io/navneetkarnam-issue-%231056/#!pages/firststeps.md

**Comment by navneetkarnam** [comment]
@sarah-lu102 ohh.. thanks a lot!

**Comment by kanishk1010** [comment]
@navneetkarnam The character '#' is represented by %23 in URL encoding

File: pages/firststeps.md [hunk 1]
```diff
@@ -95,7 +95,11 @@ Create three more issues, add comments to three other issues, and resolve three
 
 Head over to [Open Learning Exchange Repository](https://github.com/open-learning-exchange/open-learning-exchange.github.io) and make sure you have the previous list from Step 8 done.
 
-If you meet these requirements let us know in the Gitter chat, so we can send the survey to your community. Then, fill out the survey and submit it. Next, Sync your community with the nation (as you did in step 7). Lastly, once you have completed all other steps, add yourself to the virtual intern list found in team.md and create a pull request.
+If you meet these requirements let us know in the Gitter chat, so we can send the survey to your community. Then, fill out the survey and submit it. Next, Sync your community with the nation (as you did in step 7).
+
+**NOTE**: The survey sent by __@dogi__ will appear at the bottom of the page where the `Publications` and `Updates` appear in your community BeLL. Click on `Survey`, fill it out and submit it.
+
+Lastly, once you have completed all other steps, add yourself to the virtual intern list found in team.md and create a pull request.
```
Comment (kylemathias) [inline pages/firststeps.md]: @all i wish i caught this earlier, but the fix @navneetkarnam did is not technical correct.

After they complete step 8, they will have a meeting with @dogi who will then send them a survey during a "welcome" interview. During this interview, they add themselves to the team.md, and then they are sent a survey. I will make an issue about needing to make this change for accuracy.

**Comment by navneetkarnam** [comment]
@kylemathias I think the survey is usually sent before the interview because that is how it is written in the first steps page and only adding ourselves to team.md is done during the interview.

**Pull request merged**